### PR TITLE
Updating Azure Vendor dependencies

### PR DESCRIPTION
**Pull request by cfBrianMiller**
This PR updates the following packages:

github.com/Azure/azure-sdk-for-go 
 From: v9.0.1-beta To: v39.1.0
github.com/Azure/go-autorest 
 From: v7.2.1 To: v13.3.2

Issue:
when building the script with go or using the script/build the binary builds but when running make the build fails. Any advice?

### CODE REVIEW

**Comment by daxmc99** [comment]
Hi @cfBrianMiller, 
Can you drop the dep upgrades. We probably should remove that entirely now since its basically unused.

**Comment by cfBrianMiller** [comment]
@daxmc99 by removing the dep upgrades there is a list of errors about finding packages that are present. I believe the dapper build process is using the vendor directory rather than go.mod for dependency resolution but i'm not sure how to resolve that.

File: drivers/azure/azureutil/auth.go [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 
-	"github.com/rancher/machine/libmachine/log"
-	"github.com/rancher/machine/libmachine/mcnutils"
+	"github.com/docker/machine/libmachine/log"
+	"github.com/docker/machine/libmachine/mcnutils"
 
+	"github.com/Azure/go-autorest/autorest/adal"
 	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/Azure/go-autorest/autorest/to"
-	"github.com/rancher/machine/drivers/azure/logutil"
```
Comment (daxmc99) [inline drivers/azure/azureutil/auth.go]: why are we swithing to docker machine here?

**Comment by deniseschannon** [comment]
Closing in favor of https://github.com/rancher/machine/pull/72

Available as of v0.15.0-rancher36

**Pull request closed without merging**